src/scheduler/pbs_sched.cpp

```diff
@@ -835,29 +741,34 @@ wait_for_cmds()
 
 /**
  *
- * @brief sends end of cycle indication to the Server
+ * @brief sends end of cycle indication to all the servers configured
  *
- * @param[in] sconn - connection info to server
+ * @param[in] sd - socket descriptor to send end of cycle notification
  *
  * @return void
  */
 static void
-send_cycle_end(sched_svrconn *sconn)
+send_cycle_end()
 {
+	svr_conn_t *svr_conns;
+	int i;
 	static int cycle_end_marker = 0;
+	svr_conns =  static_cast<svr_conn_t *>(get_conn_svr_instances(clust_secondary_sock));
```

Comment:
Can we find a way to avoid the explicit casts? they are so ugly

Reply:
Same as the reason https://github.com/openpbs/openpbs/pull/2068#discussion_r510746731